crates/loader/src/local.rs

```diff
@@ -374,13 +374,29 @@ impl LocalLoader {
                     quoted_path(&dest_parent)
                 )
             })?;
-        crate::fs::copy(src, dest).await.with_context(|| {
-            format!(
-                "Failed to copy {} to {}",
-                quoted_path(src),
-                quoted_path(dest)
-            )
-        })?;
+
+        let failed_to_copy_error = format!(
+            "Failed to copy {} to {}",
+            quoted_path(src),
+            quoted_path(dest)
+        );
+
+        let dest_str = dest
+            .to_str()
+            .expect("Could not convert destination path to str");
```

Comment:
Better is probably `if let Some(dest_str) = dest.to_str() { /* do the check */ }`, or to use `dest.display()`.  Don't tear down the entire program just because some poxy path isn't stringable.